### ISSUE TITLE
select MeasureRange 1 fails for simple Mei file

### ISSUE THREAD

**Issue by andre2007** [rism-digital/verovio#3900]
Hi,

this issue is reproducible with Python. I installed python and release 4.4.0 of Verovio using pip.

Python sample code:
``` python
import verovio

tk = verovio.toolkit()
tk.loadFile("Bye_Bye_Love_Short.mei")
print('Hello1')

tk.select({"measureRange":"1"})
tk.redoLayout()

svg_string = tk.renderToSVG(1)
print('Hello2')
```

Veroio is crashing on tk.redoLayout. Terminal output is only "Hello1".

The example file:
[Bye_Bye_Love_Short.zip](https://github.com/user-attachments/files/18222049/Bye_Bye_Love_Short.zip)

Kind regards
André

**Comment by lpugin** [comment]
Your code works for me
<img width="639" alt="image" src="https://github.com/user-attachments/assets/3cdb7724-3ce7-43d8-a632-1179cbc4cfce" />
It also works in JavaScript.
What is the OS and the Python version?

**Comment by andre2007** [comment]
Windows 11 Home (24H2)
Python 3.12.8
Verovio Pip Version: 4.4.0

What can I do to provide you more details what is going on? I tried to set debug level 
(`verovio.enableLog(verovio.LOG_ERROR)`) and route stderr to stdout (`python test_verovio.py 2>&1`), 
but nothing is shown in console.

**Comment by andre2007** [comment]
I made an oberservation, the issue disappear, when I change this line:
`<keySig xml:id="k1sckjjd" sig="1f" />`
to
`<keySig xml:id="k1sckjjd" sig="0" />`

The original file is a Music XML file, I got the Mei
 file by the toolkit method "getMEI".

This is the original Music XML file:
[Bye_Bye_Love_Short.zip](https://github.com/user-attachments/files/18222307/Bye_Bye_Love_Short.zip)

**Comment by lpugin** [comment]
Does calling `redoLayout` also crashes without the `select`?

What can you see in the Python crash log?

**Comment by andre2007** [comment]
Thanks for having a look. Without the `select` it does not crash.

With Python crash log, you mean using a try/catch? This does not work. It seems the crash in Verovio causes the whole Python process to terminate.

**Comment by andre2007** [comment]
Thanks a lot :)

I just tried an can confirm you correction fixes my issue.

**Comment by lpugin** [comment]
There was a bug when calling `Toolkit::Select`. I am not sure why it crashed only on the Windows environment. This now fixed in today's 4.5.1 release. Presumably (and hopefully) also fixes #3624
